docs/l4-l7.rst

```diff
@@ -479,3 +515,38 @@ Dynamic Endpoint Discovery functionality is provided by F5ACIServiceCenter and h
+
+API URL: https://<BIGIP-IP:Port-Here>/mgmt/shared/appsvcs/declare?showHash=true
+Request type: GET
+Authentication type: Basic Auth (Enter username & password of the BIG-IP here)
+
+Hence, If the concerned partition is present in the API response, please report this issue/bug with the F5 BIG-IP AS3 team.  If the concerned partition is not present in the API response, please report this issue with F5 FASC team at https://github.com/F5Networks/f5-aci-servicecenter/issues
+
+------
+
+**Q.Why do I receive an error while updating the Route domain of an AS3/FAST Application?**
+
+The FASC application throws the following error message while trying to update the Route Domain from one Route Domain to another Route Domain - Error Message - “On updating the Route domain of this application, the Virtual Server of this application would reference the new Route domain. However, the Pool members of this Endpoint group may not reference the new Route domain. To continue, kindly delete this application and create a new one with the required Route domain.” This behavior was observed due to an existing bug in the AS3 Service Discovery plugin v3.41, where on moving an existing AS3 application from one Route Domain to another, the Service Discovery nodes are not updated to the new Route Domain. For example, if an application is mapped to a Tenant | Application Profile | EPG from the APIC and you are trying to update the Route Domain of this application from 0 to a non-zero Route Domain then the Virtual Server of this application will get updated with the newly assigned Route Domain. However, the EPG members from the APIC would still reference the older Route Domain inside the application Pool on the BIG-IP. Here is the link for the bug filed with AS3 Team - https://github.com/F5Networks/f5-appsvcs-extension/issues/669 . As a workaround suggested in the error message, you can delete this AS3/FAST application and create a new one with the desired Route Domain. 
+
+Note that, if the application that is deleted is the only application in that Partition, then the FASC App will attempt to delete the Partition from the BIG-IP. However, the BIG-IP does not support deleting a Partition if there are any existing objects present in it, in our case - Route Domain. Hence, the delete application operation would fail in this scenario. You are suggested to create a new application in this partition first and then delete the older application.
+
+------
+
+**Why does AS3/FAST form display Route Domains in Common partition when user selects “Create New Partition” from the Partition dropdown?**
```

Comment:
Why does the AS3/FAST form display Route Domains in the Common partition when the user selects...